app.go

```diff
@@ -148,7 +189,34 @@ func (app *App) FetchService(service *Service) error {
 			return err
 		}
 	}
+	return nil
+}
+
+func (app *App) RenderTemplates(config *ProjectConfig) error {
```

Comment:
Maybe split this also into `RenderTemplates` and `RenderTemplate` to make this better readable.